src/index.html

```diff
@@ -31,7 +31,11 @@
     <script nomodule src="/build/manifold-subscription.js"></script>
   </head>
   <body>
-    <manifold-init env="stage" client-id="3zx3qeb15z70kcv4nh8cjedhndwz2"></manifold-init>
+    <manifold-init
+      env="stage"
+      client-id="3zx3qeb15z70kcv4nh8cjedhndwz2"
+      auth-token="eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIyMWpkZmhyenVqa2U0emVldmVyMTZ6eWRjNXhteSIsInN1YiI6Im1vdGhlck9EcmFnb25zIiwiaWF0IjoiMjAyMC0wNC0xNVQxNDo0Mjo1OS40MzQ5ODIxOTdaIiwiZXhwIjoiMjAyMC0wNC0xNlQxNDo0Mjo1OS40MzQ5ODIxOTdaIiwiYXVkIjoibWFuaWZvbGQuY28vZ2F0ZWtlZXBlciIsInZhbHVlIjoiVmRxbUFUTzgiLCJwbGF0Zm9ybSI6IjN6eDc3ZHVxamZ3NGpoZXBmajMydTR2MnJubTFlIiwidG9rZW5fc2VjcmV0Ijoid2lxYlNGclVZOTE5WUN1dGhrUE42N3VxcnFTcVpLbTFzX2ZvOG1CUlhNLWFiVi1CeG42TkRleXFUVThYIn0.bq2e41Hngrq7lUSMOsSGEJwWWfUz9-vS-XUurYIGL-SuLKQUWCn_wPXoc0BkgY5SdW440L6mJXVef7H8CHI9Mw"
```

Comment:
This was good for testing (I saw it working myself), but do we want to leave this here?